lib/routes/home_route/home_route.dart

```diff
@@ -136,8 +136,7 @@ class HomeState extends State<Home> {
       final now = clock.now();
       // Show toast when user presses back button on main route, that
       // asks from user to press again to confirm that he wants to quit the app
-      if (_lastBackPressTime == null ||
```

Comment:
Running `flutter format -l 120 .` formats this line. Is that an inconsistency between Flutter on Windows vs Linux or was it just missed for #61?

Reply:
Seems like accidentally discarded this file